static/styles/left-sidebar.scss

```diff
@@ -7,10 +7,38 @@
     font-weight: 800;
 }
 
+/*
+To keep alignment consistent between the top left
+corner and the stream list, we define min widths
+together.
+*/
+#global_filters .filter-icon,
+.stream-privacy {
+    min-width: 19px;
```

Comment:
This might be a good use case for an SCSS variable that we do math with in the places that need it, rather than consolidating these all in one place?  I think that might be more robust to future refactors.

Reply:
done